Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,28 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Can this method work for both Tom Jefferson and the second person?
```

Comment:
Looks good, but second person is missing.

Reply:
Have made those changes now, let me know if this approach is ok. Thanks

Reply:
Almost, but still need to read input from console.